docs/Technical/README.md

```diff
@@ -10,59 +10,111 @@ The [Shared drive](https://drive.google.com/drive/folders/0AJ6YEVtSfOQVUk9PVA) h
 Team members who need to manage assets can be granted **Contributor** access by any of the managers.
 
 ### Google Scripts
-Periodically a [Google Script ](https://github.com/DFE-Digital/GITISContent/tree/main/Google) needs to run. This will transfer the assets from the [Shared drive](https://drive.google.com/drive/folders/0AJ6YEVtSfOQVUk9PVA) to Azure.
+Periodically a [Google Script](https://github.com/DFE-Digital/GITISContent/tree/main/Google) needs to run. This will transfer the assets from the [Shared drive](https://drive.google.com/drive/folders/0AJ6YEVtSfOQVUk9PVA) to an Azure Storage account.
 
-The script needs to have a [Azure SAS Key](https://docs.microsoft.com/en-us/azure/storage/common/storage-sas-overview) embedded into it, which gives access to the Azure Storage account. Since this key is sensistive, it is stored as a GitHub Secret and injected into the script at deployment time.
+The script needs to have an [Azure SAS Key](https://docs.microsoft.com/en-us/azure/storage/common/storage-sas-overview) embedded into it, which gives access to the Azure Storage account. Since this key is sensitive, it is stored in Azure Key Vault and injected into the script at deployment time.
 
 
 ## Azure
 ### Storage Accounts
-Within the Azure subscription s146-getintoteachingwebsite-production the [Storage Account](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-overview?toc=/azure/storage/blobs/toc.json) **s146p01gitiscontent** has been created. This storage account has a container called "**content**" which has been set as a [Static Website](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blob-static-website)
+Within the relevant Azure subscription a [Storage Account](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-overview?toc=/azure/storage/blobs/toc.json) `SubIdEnvIdgitiscontent` has been created.
 
-A [Shared Access signature](https://docs.microsoft.com/en-us/azure/storage/common/storage-sas-overview) has been created for this [Storage Account](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-overview?toc=/azure/storage/blobs/toc.json)
+```
+make environment storage_parameters
```

Comment:
`<environment>` 
Maybe also add a section to describe the 2 environments, development and production

Reply:
Useful detail: `<environment>` instead of `environment`